### PR TITLE
Added multicast support

### PR DESCRIPTION
**Pull request by OriBenHur-Any**
Now you can switch between multicast and unicast mode
by runnig the following from within the docker-compose folder:
sudo python3 ./change_mode.py

### CODE REVIEW

File: change_mode.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+        return False
+
+
+if __name__ == "__main__":
+    euid = os.geteuid()
+    if euid != 0:
+        raise EnvironmentError("need to be root")
+
+    yml_file = "/home/ori/WorkingDir/docker-compose-gpu.yml"
```
Comment (sashas-anyvision) [inline change_mode.py]: bam .. what is this path :)

File: change_mode.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    MODE = getmode()
+    re = input("Mode now is " + MODE + " change it? ")
```
Comment (sashas-anyvision) [inline change_mode.py]: please use argparse

File: change_mode.py (hunk 1 above)
Comment (OriBenHur-Any) [reply change_mode.py]: 🙄 how in god name it's here i swear that I've removed it before the commit!!

**Comment by OriBenHur-Any** [comment]
fixed it, used a different approach.
I assume that the script is in the same dir with the docker-compose so no need to get arg from the command line

File: change_mode.py (hunk 2 above)
Comment (OriBenHur-Any) [reply change_mode.py]: fixed it, used a different approach.
I assume that the script is in the same dir with the docker-compose so no need to get arg from the command line

Comment (sashas-anyvision) [reply change_mode.py]: why are you using input ? why not using argparse

Comment (OriBenHur-Any) [reply change_mode.py]: Where?
line 77?

**Comment by OriBenHur-Any** [comment]
Done.

**Comment by sashas-anyvision** [comment]
again commit reorg ..

**Pull request closed without merging**